roofit/roofitcore/src/RooAbsCachedPdf.cxx

```diff
@@ -223,12 +223,8 @@ RooAbsCachedPdf::PdfCacheElem::PdfCacheElem(const RooAbsCachedPdf& self, const R
   }
 
   // Create RooDataHist
```

Comment:
Why changing here ? String should work

Reply:
In the original version of the PR it didn't work, that's why I changed it.
When I implemented the TString compatibility, I didn't revert the change in `RooAbsCachedPdf` because it's nicer with the `std::string` anyway.

Reply:
Ok, I understand it now